_posts/it/2019-10-23-Developers_Italia_a_Codemotion_2019_ci_vediamo_a_Milano.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  Developers Italia a Codemotion 2019: ci vediamo a Milano!
+subtitle: Un booth e un meetup il 24 e 25 ottobre
+date:   2019-10-23 12:00:00 +0100
+tags: developers call
+categories: news
+author: Andrea Biancini
+image: /assets/icons/logo-it.svg
+locale: it
+ref:
+  en: 
+redirect_from:
+   -
+---
+
+Ciao a tutti!
+
+Il 24 e 25 ottobre si terrà a Milano (Superstudio, Via Tortona 27) [Codemotion 2019](https://events.codemotion.com/conferences/milan/2019/), la più grande conferenza tecnica per sviluppatori d’Italia.
+
+Come nelle precedenti edizioni, [Developers Italia](https://developers.italia.it/) parteciperà anche a quest’anno! Saremo presenti con un *booth* entrambe le giornate. Durante l’evento avremo modo di descrivere il lavoro che stiamo facendo per **supportare la trasformazione digitale  all’interno della Pubblica Amministrazione italiana** e per favorire la diffusione di software *open source*, in particolare con le attività legate al [Catalogo del Software](https://developers.italia.it/it/software/) e le [Linee Guida per l’Acquisizione e il Riuso del Software](https://developers.italia.it/it/riuso).
```

Comment:
```suggestion
Come nelle precedenti edizioni, [Developers Italia](https://developers.italia.it/) parteciperà anche a quest’anno! Saremo presenti con un *booth* entrambe le giornate. Durante l’evento avremo modo di descrivere il lavoro che stiamo facendo per **supportare la trasformazione digitale  all’interno della Pubblica Amministrazione italiana** e per favorire la diffusione di software *open source*, in particolare con le attività legate al [Catalogo del Software](https://developers.italia.it/it/software/) e le [Linee Guida per l’Acquisizione e il Riuso del Software](https://developers.italia.it/it/riuso).
```
@francescomda non mi piace molto la prima frase: "Come nelle precedenti edizioni, [Developers Italia](https://developers.italia.it/) parteciperà anche a quest’anno!", potresti rivederla leggermente? Grazie

Reply:
ok ci metto mano un secondo